src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
```

Comment:
The return value here is hard-coded as well. The function should calculate the total cost based on the daily rate and any discounts for renting 3 or more days.